javascript/attributes.js

```diff
@@ -31,23 +33,52 @@ export const extractElementAttributes = element => {
       return memo
     }, {})
 
-  attrs.value = element.value
   attrs.checked = !!element.checked
   attrs.selected = !!element.selected
   attrs.tag_name = element.tagName
-  if (element.tagName.match(/select/i)) {
-    if (element.multiple) {
-      const checkedOptions = Array.prototype.slice.call(
-        element.querySelectorAll('option:checked')
-      )
-      attrs.values = checkedOptions.map(o => o.value)
-    } else if (element.selectedIndex > -1) {
-      attrs.value = element.options[element.selectedIndex].value
+
+  if (hasMultipleOptions(element)) {
+    attrs.values = collectCheckedOptions(element)
+    delete attrs.value
```

Comment:
Wondering if we should leave value assigned (as a comma delimited list) for consistency. That would mean selects would look like this on the server.

```ruby
# single value
element[:value]  #=> 1
element[:values] #=> [1]

# multiple values
element[:value]  #=> "1,2,3"
element[:values] #=> [1,2,3]
```

Reply:
good idea! we should mention that in the docs, too